lib/nexmo_developer/app/webpacker/stylesheets/active_admin.scss

```diff
@@ -8,10 +8,14 @@
 // $sidebar-width: 242px;
 
 // Active Admin's got SASS!
-@import "active_admin/mixins";
-@import "active_admin/base";
+@import "~@activeadmin/activeadmin/src/scss/mixins";
+@import "~@activeadmin/activeadmin/src/scss/base";
 
-// Overriding any non-variable SASS must be done after the fact.
+/* Active Admin Print Stylesheet */
+@import "~@activeadmin/activeadmin/src/scss/print";
```

Comment:
this file is missing

Reply:
if the first item isn't necessary because of its inclusion in the pack, then this one is the same also